statham/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.13.5"
+__version__ = "0.13.6"
```

Comment:
```suggestion
__version__ = "0.13.5"
```

As above, the release tool will handle version bumping post-merge